### DISCUSSION TITLE
is there a binary collisional ionization feature?

### DISCUSSION THREAD

**Discussion by ZippoFight** [BLAST-WarpX/warpx#3714] [Q&A] [answered]
I am new to WarpX, and noticed ionization can be modelled by using a "background_mcc" type collision between electrons and background. I am wondering how to calculate the collisional ionization between electrons and atoms (or ions) that are not backgroud? For example between electrons and Ar1+ to produce Ar2+? Is it included in the "pairwisecoulomb" type of collision?

**Comment by ax3l** [comment, marked as answer]
Sorry for not answering here, discussion is definitely a good place to start this feature request.

Conversation on this moved to #3713, please let us know if we overlooked something here.
